tidb-resource-control.md

```diff
@@ -209,20 +210,25 @@ Runaway Queries 指那些执行时间或者消耗的资源超出预期的查询
 - `COOLDOWN`：将查询的执行优先级降到最低，查询仍旧会以低优先级继续执行，不占用其他操作的资源。
 - `KILL`：识别到的查询将被自动终止，报错 `Query execution was interrupted, identified as runaway query`。
 
-为了避免并发的 Runaway Queries 太多，在被条件识别前就将系统资源耗尽，资源管控引入了一个快速识别的机制。借助子句 `WATCH`，当某一个查询被识别为 Runaway Query 之后，在接下来的一段时间里（通过 `DURATION` 定义） ，当前 TiDB 实例会将匹配到的查询直接标记为 Runaway Query，而不再等待其被条件识别，并按照当前应对操作执行。其中 `KILL` 操作报错 `Quarantined and interrupted because of being in runaway watch list`。
+为了避免并发的 Runaway Queries 太多，在被条件识别前就将系统资源耗尽，资源管控引入了一个 Runaway Queries 监控机制，来实现快速识别和隔离。借助子句 `WATCH`，当某一个查询被识别为 Runaway Query 之后，会从这个查询提取匹配特征（通过 `WATCH` 后的匹配方式参数决定），在接下来的一段时间里（通过 `DURATION` 定义），这个 Runaway Query 的匹配特征会被加入监控列表，TiDB 实例会将查询和监控列表进行匹配，匹配到的查询直接标记为 Runaway Query，而不再等待其被条件识别，并按照当前应对操作进行隔离。其中 `KILL` 会结束该查询，并报错 `Quarantined and interrupted because of being in runaway watch list`。
```

Comment:
```suggestion
为了避免并发的 Runaway Queries 太多，在被条件识别前就将系统资源耗尽，资源管控引入了一个 Runaway Queries 监控机制，来实现快速识别和隔离。借助子句 `WATCH`，当某一个查询被识别为 Runaway Query 之后，会从这个查询提取匹配特征（通过 `WATCH` 后的匹配方式参数决定），在接下来的一段时间里（通过 `DURATION` 定义），这个 Runaway Query 的匹配特征会被加入监控列表，TiDB 实例会将查询和监控列表进行匹配，匹配到的查询直接标记为 Runaway Query，而不再等待其被条件识别，并按照当前应对操作进行隔离。其中 `KILL` 会终止该查询，并报错 `Quarantined and interrupted because of being in runaway watch list`。
```